src/proto/media/file/mod.rs

```diff
@@ -214,6 +214,16 @@ pub trait File: Sized {
         // global allocator.
         unsafe { Ok(Box::from_raw(info)) }
     }
+
```

Comment:
I'm not sure if we want to have this on the File trait or only on the FileHandle abstraction. At first glance, I think, having it in the trait might be the better option